### PR TITLE
Autostart doctoc for all Readme on pre-commit

### PR DESCRIPTION
**Pull request by qWici**
I saw that work on the PR #182 stopped, so I did it myself. 
Now contributors must run only 1 command - 'npm i'.

### CODE REVIEW

File: bin/deploy-translation.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+var doctoc = require("doctoc");
+var shell = require("shelljs");
+var glob = require("glob");
+
+/*
+    Generate menu for all ReadMe with DocToc
+ */
+glob("./README*.md", function (er, files) {
+    files.forEach(function (item) {
+        shell.exec("doctoc " + item + " --github");
```
Comment (sotayamashita) [inline bin/deploy-translation.js]: What about to change code like `doctoc *.md --github`

File: bin/deploy-translation.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#! /usr/bin/env node
+var doctoc = require("doctoc");
+var shell = require("shelljs");
+var glob = require("glob");
```
Comment (sotayamashita) [inline bin/deploy-translation.js]: Please use `const` instead of `var`

File: bin/deploy-translation.js (hunk 1 above)
Comment (qWici) [reply bin/deploy-translation.js]: I thought about it. But the repository may also other .md files (like CONTRIBUTE.md) that do not need to be processed with DocToc.

**Comment by buslov** [comment]
Generated links may be incorrect. doctoc depends on anchor-markdown-header that (<= 0.5.7) has [bug with non-ASCII headers](https://github.com/thlorenz/anchor-markdown-header/pull/39) 
Known issues: [issue 97](https://github.com/thlorenz/doctoc/issues/97), [issue 122](https://github.com/thlorenz/doctoc/issues/122)

**Comment by RichardLitt** [comment]
I think you need to add a .gitignore with node_modules, and check in the package-lock.json. 

Adding a `## Testing` heading below the TOC didn't seem to work. Can you test this for me?

**Comment by RichardLitt** [comment]
Closing due to https://github.com/k88hudson/git-flight-rules/pull/182#issuecomment-388885360. Thank you, all.

**Pull request closed without merging**